schema/proto3/dhcp.proto

```diff
@@ -27,10 +27,12 @@ message DHCPConfiguration {
     uint32 format_version = 1;
     uint32 revision_number = 2;
 
-    string mac_address = 3;
-    string ipv4_address = 4;
-    string ipv6_address = 5;
-    string port_host_name = 6; // for local DNS response
+    string request_id = 3;
+    string subnet_id = 4;
```

Comment:
If we need to merge the dhcp change earlier, we could create a separate PR for that.

Reply:
ok, I don't think dhcp progress is blocked because they will be using a random/fixed server address for DHCP server for now.